packages/firestore/src/remote/bloom_filter.ts

```diff
@@ -75,6 +75,10 @@ export class BloomFilter {
     this.bitCountInInteger = Integer.fromNumber(this.bitCount);
   }
 
+  isEmpty(): boolean {
```

Comment:
Please remove the isEmpty() method, and just use bitCount directly.

The problem is that container classes typically use a method named "isEmpty()" as a shorthand for "size==0". But the concept of "size" on a bloom filter is ambiguous: is it the number of bytes in the underlying byte array, the number of bits of the underlying byte array that are used, or the number of values that the bloom filter contains (which is undefined) .